backend/Pipfile

```diff
@@ -12,6 +12,8 @@ requests-mock = "*"
 [packages]
 requests = "*"
 flask = "*"
+pm4py = "*"
+atomicwrites = "*"
```

Comment:
Do we still need to explicitly include atomicwrites even after fixing the issues with your shell?